mlflow/deployments/utils.py

```diff
@@ -80,6 +81,9 @@ def get_deployments_target() -> str:
         return _deployments_target
     elif uri := MLFLOW_DEPLOYMENTS_TARGET.get():
         return uri
+    elif is_in_databricks_runtime():
```

Comment:
Do we set the `MLFLOW_DEPLOYMENTS_TARGET` environment variable on Databricks? I remember we do.

Reply:
Ah it seems we set it since 3 months ago, then we can remove this default. Nevertheless, I think we shouldn't return None silently..?

Reply:
yeah returning None is confusing.